src/resources/org/apache/cassandra/cql3/reserved_keywords.txt

```diff
@@ -54,4 +54,5 @@ USE
 USING
 VIEW
 WHERE
-WITH
\ No newline at end of file
+WITH
```

Comment:
We are missing more than these two. GENERATED, LIKE ... Probably deserves its own ticket.